modules/tracking/samples/lasot_benchmark.py

```diff
@@ -0,0 +1,187 @@
+        elif tracker_name == "MedianFlow":
+            tracker = cv.TrackerMedianFlow_create()
+        elif tracker_name == "GOTURN":
+            tracker = cv.TrackerGOTURN_create()
+        elif tracker_name == "MOSSE":
+            tracker = cv.TrackerMOSSE_create()
+        elif tracker_name == "CSRT":
+            tracker = cv.TrackerCSRT_create()
+
+        init_once = False
+
+        print("Video: " + str(video_name))
+
+        video_name = video_name.replace("\n", "")
+        gt_file = open(
+            args.path_to_dataset + video_name + "/groundtruth.txt", "r")
+        gt_bb = gt_file.readline().replace("\n", "").split(",")
+        init_bb = gt_bb
+
+        for _ in range(len(init_bb)):
+            init_bb[_] = float(init_bb[_])
+
+        init_bb = tuple(init_bb)
+
+        print("Initial bounding box: ", init_bb)
+
+        video_sequence = sorted(
+            glob.glob(str(args.path_to_dataset + str(video_name) + "/img/*.jpg")))
+
+        print("Number of frames in video: " + str(len(video_sequence)))
+
+        sum_iou = 0
+        sum_pr = 0
+        sum_norm_pr = 0
+        frame_counter = len(video_sequence)
+
+        #For every frame in video
+        for number_of_the_frame, image in enumerate(video_sequence):
+
+            for _ in range(len(gt_bb)):
+                gt_bb[_] = float(gt_bb[_])
+            gt_bb = tuple(gt_bb)
+
+            frame = cv.imread(image)
+
+            #Re-initializing rate in frames
+            if tracker_name == "Boosting":
+                frames_before_reinit = 500
+            elif tracker_name == "MIL":
+                frames_before_reinit = 1000
+            elif tracker_name == "KCF":
+                frames_before_reinit = 110
+            elif tracker_name == "MedianFlow":
+                frames_before_reinit = 20
+            elif tracker_name == "GOTURN":
+                frames_before_reinit = 20
+            elif tracker_name == "MOSSE":
+                frames_before_reinit = 20
+            elif tracker_name == "CSRT":
+                frames_before_reinit = 250
+
+            #Initialization and re-initialization of tracker
+            if ((number_of_the_frame + 1) % frames_before_reinit == 0) and (
+                number_of_the_frame != 0):
+
+                if tracker_name == "Boosting":
+                    tracker = cv.TrackerBoosting_create()
+                elif tracker_name == "MIL":
+                    tracker = cv.TrackerMIL_create()
+                elif tracker_name == "KCF":
+                    tracker = cv.TrackerKCF_create()
+                elif tracker_name == "MedianFlow":
+                    tracker = cv.TrackerMedianFlow_create()
+                elif tracker_name == "GOTURN":
+                    tracker = cv.TrackerGOTURN_create()
+                elif tracker_name == "MOSSE":
+                    tracker = cv.TrackerMOSSE_create()
+                elif tracker_name == "CSRT":
+                    tracker = cv.TrackerCSRT_create()
```

Comment:
please remove code duplication. In example, define a method which returns tracker by name

Reply:
Added new method for initialization and re-initialization of tracker, which also return number of frames for re-initialization.